spec/requests/notifications_spec.rb

```diff
@@ -28,15 +27,18 @@
           end
 
           context 'and the requested format is html' do
-            let(:format) { 'html' }
+            let(:headers) { { 'HTTP_REFERER' => previous_page } }
 
             it 'redirects the user back to where they were before' do
               expect(response).to redirect_to previous_page
             end
           end
 
           context 'and the requested format is json' do
-            let(:format) { 'json' }
+            let(:headers) do
+              { 'HTTP_REFERER' => previous_page,
+                'ACCEPT' => 'application/json' }
+            end
```

Comment:
I'm personally not a fan of this code format
```suggestion
            let(:headers) do
              { 
                'HTTP_REFERER' => previous_page,
                'ACCEPT' => 'application/json' 
              }
            end
```

Reply:
Formatted the headers.